### DISCUSSION TITLE
能否考虑把数字英文中文单独做一个字体包？

### DISCUSSION THREAD

**Discussion by Jevon6666** [atelier-anchor/smiley-sans#85] [General]
能否考虑把数字英文中文单独做一个字体包？让其专门用于统计数值展示？整体字体包现在比较大，不适合在网页及小程序中引用。

**Comment by sunjianan9900** [comment]
你可以使用字体裁剪的工具自己裁剪一个出来哦，比如 http://font-subset.disidu.com/ 这种
